products/memgraph/upcoming/tutorials/got-deaths.md

```diff
@@ -0,0 +1,232 @@
+  memgraph --data-directory /usr/share/memgraph/examples/Got \
+  --storage-snapshot-interval-sec=0 --storage-wal-enabled=false \
+  --storage-snapshot-on-exit=false --storage-properties-on-edges=true
+```
+
+The user should note that any modifications of the database state will persist
+only during this run of Memgraph.
+
+### Example Queries using OpenCypher
+
+In the queries below, we are, as usual, using [OpenCypher](https://www.opencypher.org)
+to query Memgraph via the console.
+
+Here are some queries you might find interesting:
+
+**MINI-GAME** - If you have watched the TV Show, try to guess each result before executing the query!
+1) Let's start with a couple of simple queries. List the locations with the most deaths occurred.
+Can you guess which one is it?
+
+```opencypher
+MATCH (l:Location)<-[:InLocation]-(d:Death) 
+return l.name AS LocationName, COUNT(d) AS DeathCount
+ORDER BY DeathCount DESC
+```
+
+2) Now that we have the location with the most deaths, let's list the episodes with the most deaths
+as well.
+
+```opencypher
+MATCH (d:Death)-[:InEpisode]->(e:Episode)
+RETURN e.name, COUNT(d) AS KillCount
+ORDER BY KillCount DESC
+```
+
+3) List the number of kills per season. I think we all know the highest count one. 
+
+```opencypher
+MATCH (d:Death)-[:InSeason]->(s:Season) 
+RETURN s.number AS SeasonNumber, COUNT(d) as DeathNumber
+ORDER BY SeasonNumber ASC
+```
+
+4) While we know which season was by far the worst, can you guess the best one based on user ratings?
+Let's list the seasons by IMDB ratings. The problem we get with using the `AVG()` function is that it
+gives us too many decimals, therefore a useful solution is given in this example using `ROUND()`.
+
+```opencypher
+MATCH (e:Episode)-[:InSeason]->(s:Season) 
+RETURN s.number AS SeasonName, ROUND(100 * AVG(e.imdb_rating))/100 AS Rating
+ORDER BY Rating DESC
+```
+
+5) There are many methods characters died with such as with sword or Dragonfire, but let's list victims
+of unique methods.
+
+```opencypher
+MATCH (c:Character)-[k:Killed]->(c2:Character)
+WITH k.method AS Method, COUNT(k.method) AS MethodCount
+WHERE MethodCount < 2
+MATCH (c:Character)-[k:Killed]->(c2:Character)
+WHERE k.method = Method
+RETURN Method, c2.name as Victim
+```
+
+6) Daenerys Stormborn of House Targaryen, the First of Her Name, Queen of the Andals and the First Men,
+Protector of the Seven Kingdoms, the Mother of Dragons, the Khaleesi of the Great Grass Sea, the Unburnt,
+the Breaker of Chains or shortened to "Daenerys Targaryen" in our database is by far the biggest killer on
+the show. Let's list all the episodes she killed in as well as characters she killed.
+
+```opencypher
+MATCH (c:Character {name: 'Daenerys Targaryen'})-[k:Killed]->(c2:Character)
+MATCH (c)-[:KillerIn]->(d:Death)<-[:VictimIn]-(c2)
+MATCH (d)-[:InEpisode]-(e:Episode)
+return distinct(c2.name) as Victim, COUNT(d) as Kills, e.name

```

Comment:
return and distinct are lowercased here while all keywords in the tutorial and UPPERCASE. Make things consistent. `e.name` - You always did AS except now, why is that missing? Consistency again.